submitted_models/robotika_x2_sensor_config_1/launch/spawner.rb

```diff
@@ -19,6 +19,10 @@ def spawner(_name, _modelURI, _worldName, _x, _y, _z, _roll, _pitch, _yaw)
   "      <wheel_separation>#{0.33559 * 1.23}</wheel_separation>\n"\
   "      <wheel_radius>0.098</wheel_radius>\n"\
   "      <topic>/model/#{_name}/cmd_vel_relay</topic>\n"\
+  "      <min_velocity>-1</min_velocity>\n"\
```

Comment:
Just checking, is this change from a `robotika` model on purpose?

Reply:
No, it's not. We can ignore this. Sorry about that.

Reply:
I thought this was requested inside the model_pr_426-> master pull request? Perhaps it was by mistake?
![robotica_x2](https://user-images.githubusercontent.com/25831722/84922879-b6111a80-b083-11ea-8638-6169ab2acfe9.png)

Reply:
Yes, sorry for the confusion, I copied this comment from a different team.